prime-dotnet-webapi/Migrations/ApiDbContextModelSnapshot.cs

```diff
@@ -12059,60 +12059,6 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.ToTable("PharmanetTransactionLog");
                 });
 
-            modelBuilder.Entity("Prime.Models.PharmanetTransactionLogTemp", b =>
```

Comment:
Shouldn't `PharmanetTransactionLogTemp` remain in `ApiDbContextModelSnapshot`?

Reply:
Good catch. Not sure how it got removed from the snapshot. I added it back manually. It should work fine now.